runtime/doc/api.txt

```diff
@@ -700,6 +700,22 @@ nvim_chan_send({chan}, {data})                              *nvim_chan_send()*
       • {chan}  id of the channel
       • {data}  data to write. 8-bit clean: can contain NUL bytes.
 
+                                                    *nvim_complete_set_info()*
+nvim_complete_set_info({index}, {text}, {opts})
```

Comment:
w.r.t the previous discussion, one change we _could_ make to make the API more future proof is to rename it `nvim_complete_set` and move `"info"` as the currently _only_ key of a typed `Dict(complete_set)` . Then it could be extended in the future if needed.